packages/plugin-node-uncaught-exception/uncaught-exception.js

```diff
@@ -1,15 +1,14 @@
-const createEventFromErr = require('@bugsnag/core/lib/event-from-error')
-
 let _handler
 module.exports = {
   init: client => {
     if (!client._config.autoDetectErrors) return
     _handler = err => {
-      client.notify(createEventFromErr(err, {
+      const event = client.BugsnagEvent.create(err, true, {
```

Comment:
Following on from https://github.com/bugsnag/bugsnag-js/pull/664/files#r354934957, why do we tolerate non errors here but not in `plugin-node-unhandled-rejection` and `plugin-window-unhandled-rejection`?

Reply:
I've updated this – it should be the same as the ones you mentioned.